src/paymentservice/src/test/java/ca/bc/gov/open/pssg/rsbc/dps/paymentservice/PaymentserviceApplicationTests.java

```diff
@@ -1,32 +1,20 @@
 package ca.bc.gov.open.pssg.rsbc.dps.paymentservice;
 
-import ca.bc.gov.open.pssg.rsbc.dps.paymentservice.types.SinglePaymentRequest;
+import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.Test;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.beans.factory.annotation.Value;
 import org.springframework.boot.test.context.SpringBootTest;
 import org.springframework.boot.test.context.SpringBootTest.WebEnvironment;
 import org.springframework.boot.test.web.client.TestRestTemplate;
 import org.springframework.boot.web.server.LocalServerPort;
 import org.springframework.test.context.TestPropertySource;
 
-import java.net.URL;
-
 import static org.assertj.core.api.Assertions.assertThat;
 
 @SpringBootTest(webEnvironment = WebEnvironment.RANDOM_PORT)
 @TestPropertySource(locations="classpath:test.properties")
```

Comment:
Do we need to define the test.properties file since it doesn't look like we're using any of it's properties anymore?

Reply:
yeah I can change that

Reply:
I guess we need, it brings also the properties from the app, I think the @TestPropertySource also gets the data from the app properties

Reply:
Ok. Guess we keep it :-)